health/README.md

```diff
@@ -190,6 +190,11 @@ Everything is the same with [badges](../web/api/badges/). In short:
      have spaces in their names). This accepts Netdata simple patterns and the `match-ids` and
      `match-names` options affect the searches for dimensions.
 
+- `foreach DIMENSIONS` is optional and it will always be the last parameter. The rules for dimension here follow the same
```

Comment:
These comments are very confusing. Please get help from @joelhans or @prhomhyse to improve the content here.

Reply:
Here is my recommendation for altering the grammar/structure of this paragraph. This one sentence—`It is used to set the group of the dimensions of a chart that will have the same rule to raise an alarm.`—is difficult for me to understand and I may have rewritten it wrong. Let's make sure it's right before any changes are made.

> `foreach DIMENSIONS` is optional, will always be the last parameter, and uses the same `,`/`|` rules as the `of` parameter. Each dimension you specify in `foreach` will use the same rule to trigger an alarm. If you set both `of` and `foreach`, Netdata will ignore the `of` parameter and replace it with one of the dimensions you gave to `foreach`.

I'm having trouble understanding how I would try to format this line if I were going to use each of these parameters. @thiagoftsm maybe you could write an example template using `lookup` and `foreach` and we could document what each parameter accomplishes? We could add this to the **Examples** section, perhaps. It may also just help me better understand how to best document this rule.

Reply:
Hi @joelhans ,

I copied and paste your suggestion in the documentation, I also created two examples in the section cited, in the examples I decided to focus in the `lookup` line that is the greatest difference from the others, but case you find that is good to describe each line I can do.
Thank you to help me here!